code/modules/power/singularity/singularity.dm

```diff
@@ -117,8 +117,18 @@
 	return
 
 
-/obj/singularity/bullet_act(obj/item/projectile/P)
-	qdel(P)
+/obj/singularity/bullet_act(obj/item/projectile/bullet/SRN_rocket/N, obj/item/projectile/P, mob/living/user)
```

Comment:
The proc itself shouldn't get the SRN rocket variable.
Put that after a type check for the rocket.